go.mod

```diff
@@ -150,3 +149,9 @@ replace (
 	k8s.io/code-generator => k8s.io/code-generator v0.24.3
 	k8s.io/component-base => k8s.io/component-base v0.24.3
 )
+
+// workaround for https://github.com/gardener/hvpa-controller/issues/92, remove once it's fixed
+replace (
+	github.com/gardener/hvpa-controller => github.com/gardener/hvpa-controller v0.4.0
+	github.com/gardener/hvpa-controller/api => github.com/gardener/hvpa-controller/api v0.4.0
+)
```

Comment:
Is this really required?
If I remember correctly, this was removed from other extensions when gardener 1.52 was revendored. The networking problem detector already has gardener 1.52.

Reply:
Good catch. This was reintroduced by the merge after rebasing.

Reply:
fixed